src/main/database/sqlite-db.ts

```diff
@@ -19,7 +19,7 @@ class SQLiteDB {
       info('Connecting to SQLite database');
       await this.connection.authenticate();
       await this.initTables();
-      await this.connection.sync();
+      await this.connection.sync({ force: true });
```

Comment:
Is `{force: true}` needed? Based on what I see online, it drops the tables before creating them each time, which means the data would always be reset on startup.

Reply:
`alter: true` is broken and doesn't work. `force: true` is not recommended for production, but creates the column I need. We need to use migrations at some point.

Reply:
Alright, fair.

Reply:
Steps to reproduce broken `alter: true`:
1. Stop any dev run process
2. Switch to main branch
3. Delete your `rbox-data.db` (if you know where its located)
4. Start a `npm run start` in the main branch
5. Stop the dev process
6. Switch to this branch
7. Change `{ force: true }` to `{ alter: true }`
8. Start a `npm run start` in the main branch
9. See errors in terminal